lib/Rest/GeneralRequest.h

```diff
@@ -133,16 +117,9 @@ class GeneralRequest {
   void setRequestType(RequestType type) { _type = type; }
 
   std::string const& fullUrl() const { return _fullUrl; }
```

Comment:
I am unclear if `_fullUrl` is still needed and/or populated. I think this PR removes it partially... 
Do we still populate it correctly and need it?

Reply:
yes it is still used